src/store/actions/actions-model.ts

```diff
@@ -57,47 +68,47 @@ export const setCertifyStatementResource: ActionIF = ({ commit }, certifyStateme
   commit('mutateCertifyStatementResource', certifyStatementResource)
 }
 
-export const setCertifyState: ActionIF = ({ commit }, certifyState): void => {
+export const setCertifyState: ActionIF = ({ commit }, certifyState: CertifyIF): void => {
   commit('mutateCertifyState', certifyState)
 }
 
 export const setCertifyStateValidity: ActionIF = ({ commit }, validity): void => {
   commit('mutateCertifyStateValidity', validity)
 }
 
-export const setBusinessContact: ActionIF = ({ commit }, businessContact): void => {
+export const setBusinessContact: ActionIF = ({ commit }, businessContact: ContactPointIF): void => {
```

Comment:
Adding types here is actually pretty useless since, in the components where we use these setters/actions, we type them all as `ActionBindingIF` and we therefore lose the formal parameter type. Oh well. :/